### ISSUE TITLE
add scp option

### ISSUE THREAD

**Issue by deitch** [lf-edge/eden#230]
We should add `eden eve scp` in addition to the `eden eve ssh`. We often need to pull files from the host; this makes it much easier.

**Comment by giggsoff** [comment]
But it needs to install scp inside EVE.

**Comment by deitch** [comment]
It can add it if needed. But you're right, let's add it to pillar. It's useful.

**Comment by rvs** [comment]
I don't think it should be in pillar by default -- arguably it can be added to sshd container but that would require a roundtrip with linuxkit maintainers.

**Comment by deitch** [comment]
I will make that happen. Let's leave this open until then.

**Comment by deitch** [comment]
This is being fixed via https://github.com/lf-edge/eve/pull/1546 
Once that is in, we can close this.

**Comment by rvs** [comment]
Actually I don't think it'll be fixed -- but that's a separate conversation ;-)
